config/acme/config_files.xml

```diff
@@ -144,30 +144,6 @@
     <desc>file containing specification of archive files for each component (for documentation only - DO NOT EDIT)</desc>
     <schema>$CIMEROOT/config/xml_schemas/config_archive.xsd</schema>
   </entry>
-  <entry id="ARCHIVE_SPEC_FILE">
```

Comment:
@agsalin please look at this. Clearly there was a mistake in the merge. I deleted the ARCHIVE_SPEC_FILE that looked like it didn't have your directory renames, but maybe there was some change here that needed to be preserved?